### PR TITLE
Context path changes does not support in Status Dashboard.

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 1]
```diff
@@ -80,10 +80,10 @@
     private  QueryManager metricsQueryManager;
     public StatusDashboardMetricsDBHandler() {
         Connection conn = null;
-        dataSource = DashboardDataHolder.getInstance().getMetricsDataSource();
+        dataSource = MonitoringDataHolder.getInstance().getMetricsDataSource();
         if (dataSource != null) {
             try {
-                conn= DashboardDataHolder.getInstance().getMetricsDataSource().getConnection();
+                conn= MonitoringDataHolder.getInstance().getMetricsDataSource().getConnection();
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: @Yasara123 space before = is missing

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/internal/services/PermissionGrantServiceComponent.java [hunk 2]
```diff
@@ -64,10 +64,10 @@ protected void start(BundleContext bundleContext) {
             logger.info("Status dashboard permission grant service component is activated.");
         }
         try {
-            DeploymentConfigs resolvedConfiguration=DashboardDataHolder.getInstance()
+            DeploymentConfigs resolvedConfiguration=MonitoringDataHolder.getInstance()
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/internal/services/PermissionGrantServiceComponent.java]: space after and before =

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java (hunk 1 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Fixed in 44e07c5322adee1cb3927ef2ae42d41caf1c3130

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/internal/services/PermissionGrantServiceComponent.java (hunk 2 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/internal/services/PermissionGrantServiceComponent.java]: Fixed in ea04a21de7eed189e8627127fcd8e4c9c600b999

**Pull request merged**